druid-shell/src/backend/windows/window.rs

```diff
@@ -1369,8 +1380,9 @@ impl WindowBuilder {
                         dwExStyle = 0;
                     }
                     WindowLevel::Modal => {
-                        dwStyle = WS_OVERLAPPED;
-                        dwExStyle = WS_EX_TOPMOST;
+                        dwStyle = WS_POPUPWINDOW | WS_CAPTION;
```

Comment:
Why would a modal be a `WS_POPUPWINDOW`? According to [this](https://docs.microsoft.com/en-us/windows/win32/winmsg/window-features#pop-up-windows) a modal window "appear outside an application's main window". In my mind, a modal window is a window that is at the **center** of the current foreground window while other windows are dimmed.

Reply:
What Im trying to say is it should be a `WS_CHILD` window. Maybe you tried it already and didnt work?

Reply:
`WS_CHILD` has many restrictions, which are not really needed. For example I don't think a modal window should be confined by its parent.

But you are right `WS_POPUPWINDOW` is also not the right choice here. We should properly use something like this `WS_OVERLAPPED | WS_CAPTION | WS_SYSMENU | WS_THICKFRAME`  and also add an option to enable the minimize and maximize buttons.

Reply:
Hmm....I have a different image of modals in my head then. I _thinnnnnkkk_ maybe @cmyr can help us here?? He said he was busy with other things on Zulip but I hope he sees this and nudges us in the right direction.